### PR TITLE
Update creating_a_reference_data.rst

### PR DESCRIPTION
**Pull request by LaureBro**
remove the part on the ProductValue class since it does not exist anymore on 2.0

### CODE REVIEW

**Comment by LaureBro** [comment]
hello dear colleagues @jmleroux @jjanvier can you review my PR pliz?

File: manipulate_pim_data/catalog_structure/creating_a_reference_data.rst [hunk 1]
```diff
@@ -174,18 +174,6 @@ To check if the entities are correctly set up, use the following command:
...
-Overriding the ProductValue
----------------------------
-
-Depending on the needs, a product can be linked to several colors or just one.
-The first case will be called *simple reference data*, while the second one will be referred to as *multiple reference data*.
-
-To link a reference data to the product, the `ProductValue` object needs to be overriden.
-This task is documented here :doc:`overriding_the_orm_product_value`.
-
-Don't forget to check the mapping of the product value and to register the custom class in the container.
```
Comment (jmleroux) [inline manipulate_pim_data/catalog_structure/creating_a_reference_data.rst]: You can keep this part.

File: manipulate_pim_data/catalog_structure/creating_a_reference_data.rst [hunk 2]
```diff
@@ -174,18 +174,6 @@ To check if the entities are correctly set up, use the following command:
     php bin/console doctrine:mapping:info
 
 
-Overriding the ProductValue
----------------------------
-
-Depending on the needs, a product can be linked to several colors or just one.
```
Comment (jmleroux) [inline manipulate_pim_data/catalog_structure/creating_a_reference_data.rst]: Line 180 and 181 can be kept to better understand the rest of this doc.

File: manipulate_pim_data/catalog_structure/creating_a_reference_data.rst [hunk 3]
```diff
@@ -174,18 +174,6 @@ To check if the entities are correctly set up, use the following command:
     php bin/console doctrine:mapping:info
 
 
-Overriding the ProductValue
```
Comment (jmleroux) [inline manipulate_pim_data/catalog_structure/creating_a_reference_data.rst]: Can be changed by `Configuring the Product Value`

**Comment by LaureBro** [comment]
Thanks @jmleroux, I've made another PR: https://github.com/akeneo/pim-docs/pull/734
Seems easier for me :)
I'll close this one.

File: manipulate_pim_data/catalog_structure/creating_a_reference_data.rst (hunk 1 above)
Comment (SamirBoulil) [reply manipulate_pim_data/catalog_structure/creating_a_reference_data.rst]: ah ok

**Pull request closed without merging**